src/backend/compiler/compiler.cc

```diff
@@ -990,38 +991,69 @@ extern "C" void seashell_preprocessor_free(struct seashell_preprocessor *preproc
 }
 
 /**
- * seashell_preprocessor_set_main_file(struct seashell_preprocessor* preprocessor, const char* file)
- * Sets the main file to begin running the preprocessor on
+ * seashell_preprocessor_set_question_dir(struct seashell_preprocessor* preprocessor, const char *dir)
```

Comment:
Fair enough.  I'm not entirely sure if we do need to handle that case (as we can simply ship the .h file in the directory where the .c file is supposed to be implemented) but let's find out tomorrow at the meeting.